index.md

```diff
@@ -1,4 +1,222 @@
+
+### Preferred Tool
+- [Dart](https://aptrust.github.io/dart-docs/) is the tool preferred by the Rockefeller Archive Center to package digital and deliver digital transfers. DART can be configured by importing settings files provided by the RAC.
+
+## Digital Transfer Structure 
+This section describes the structure of digital transfers, which allow those packages to be validated and verified. 
+Donor organizations are responsible for creating structured bags which comply with this specification and transferring them via agreed-upon protocols and schedules. 
+
+
+### Specifications 
+- Rockefeller Archive Center bags must conform to the [BagIt packaging specification](https://datatracker.ietf.org/doc/html/draft-kunze-bagit-14). 
+- Rockefeller Archive Center bags must be serialized (single .zip, .tar or tar.gz file) 
+- All bags must be valid according to the organization’s BagIt Profile. 
+
+### Structure Overview 
+This section includes a simple example of a Rockefeller Archive Center BagIt Specification-compliant bag.  
+
+
+ 
+ 
+RAC-BAG-ID/  
+
+&emsp;&emsp;    | bagit.txt  
+&emsp;&emsp;    | manifest-sha256.txt  
+&emsp;&emsp;    | bag-info.txt  
+&emsp;&emsp;    \--- data/  
+&emsp;&emsp;&emsp;&emsp;          | [payload files]  
+&emsp;&emsp;&emsp;&emsp;          | metadata.json 
+
+### Structure Components
+#### RAC-BAG-ID 
+The name of the root directory of the bag. This directory name may include Unicode characters and characters in the extended character set (128–255), except for the following reserved characters: 
+
+- < (less than) 
+- \> (greater than) 
+- : (colon) 
+- " (double quote) 
+- / (forward slash) 
+- \ (backslash) 
+- \| (vertical bar or pipe) 
+- ? (question mark) 
+- \* (asterisk) 
+
+#### bagit.txt 
+
+This file is required in the BagIt spec and includes BagIt version and tag file character encoding. 
+
+#### manifest-sha256.txt
+
+This file is required in the BagIt spec and contains a checksum for every item included in the bag’s payload. In this example we have chosen sha256, but using sha512 is also acceptable. md5 is not accepted as a checksum algorithm. 
+
+#### bag-info.txt 
+
+This file contains metadata elements, which will be indexed as structured data. For more details on the metadata fields, their content, requirements, and usage, please see the Metadata Field Specification section below. Please use standardized names and avoid the use of all acronyms as separate stakeholders may use the same acronyms for different concepts. 
+
+#### data 
+
+Required directory for payload items.  
+
+#### metadata.json 
+
+Valid JSON or JSON-LD file that includes metadata elements included in `bag-info.txt` as well as any additional elements donors wish to provide to the RAC. This file is optional. 
+
+ 
+
+## Metadata Field Specifications 
+
+ 
+
```

Comment:
Again, a bunch of empty lines which can be removed. I think we only need one line before and after a heading.